packages/dashboard/src/api/hooks/statistics/useItemStats.ts

```diff
@@ -0,0 +1,10 @@
+import { useAuthContext, useQuery } from '@openmsupply-client/common';
```

Comment:
slightly overboard with the dashboard, since it's so small. I expect more will be added to it though, so thought it worth putting the pattern in place.

The idea is that each hook has its own file, so you can CMD+P to find it easily, but then is grouped up into functional areas, and then into an overall API area. Makes the usage consistent and you only have to import the top level item (e.g. `useDashboard`)